internal/p2p/downloads.go

```diff
@@ -12,9 +12,15 @@ func ProcessDownload(metadata store.BaseMetadata) (bool, error) {
 	var fresult bool
 	if appconfInstance.SecurityCapability > 0 {
 		fmt.Println("[INFO] File Security checking CID: " + metadata.IPFSCID + " with SHA256: " + metadata.FileSHA256)
-		// TODO: 				if appconfInstance.SecurityCapability [1 2 3 4]
 
-		if appconfInstance.SecurityCapability <= 3 {
+		// Skip all security scanning if FFSkipVT is enabled
+		if appconfInstance.FFSkipVT {
+			fmt.Println("[INFO] Virus scanning disabled (FFSkipVT=true), skipping security check")
+			fmt.Println("[INFO] Fetching CID: " + metadata.IPFSCID)
+			psfs.GetFileIPFS(metadata.IPFSCID, appconfInstance.CacheFolder+"/"+metadata.IPFSCID+"."+metadata.FileType)
+			fresult = true
+		} else if appconfInstance.SecurityCapability <= 3 {
```

Comment:
Let's consider replacing these (all in this logical expression) magic numbers with an enum const.